platform/fetch/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@affinidi/platform-fetch",
+  "version": "1.0.0",
+  "description": "SDK core monorepo for affinity DID solution",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "scripts": {
+    "build": "npm run tsc",
+    "buildDocs": "typedoc --out docs src",
+    "start": "npm run build && node dist/index.js",
+    "tsc": "tsc -p tsconfig-build.json",
+    "lint": "eslint src/** test/**",
+    "checkLicenses": "license-checker --production --failOn 'AGPL 3.0;LGPL 2.1' --summary"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/affinityproject/affinidi-core-sdk.git",
+    "directory": "sdk/core"
+  },
+  "keywords": [
+    "sdk",
+    "affinity",
+    "network",
+    "did"
+  ],
+  "author": "Denis Popov <denis@slatestudio.com>",
```

Comment:
Lets use correct emails

Reply:
Better to create a separate PR to update it everywhere or even change to "author": "Affinidi", (with respect to original authors, but let's highlight we are team not individual devs)